eth/protocols/eth/handler_test.go

```diff
@@ -134,266 +130,6 @@ func (b *testBackend) GetBlockHashesFromHash(tx kv.Tx, hash common.Hash, max uin
 	return chain
 }
 
-// Tests that block headers can be retrieved from a remote chain based on user queries.
-func TestGetBlockHeaders64(t *testing.T) { testGetBlockHeaders(t, 64) }
```

Comment:
Does it make sense to modify this test to eth66?

Reply:
I am kind of wondering why it worked to be honest

Reply:
is that not ETH64?